File: homepage/src/main/java/com/hutcwp/homepage/HomeActivity.java
package com.hutcwp.homepage;

import android.os.Bundle;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.alibaba.android.arouter.launcher.ARouter;

import androidx.appcompat.app.AppCompatActivity;

@Route(path = "/homepage/home")
public class HomeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);

//        ARouter.getInstance().build("/cartoon/comic").navigation();
//        ARouter.getInstance().build("/cartoon/load").navigation();
//        ARouter.getInstance().build("/comic/list").navigation();
//        ARouter.getInstance().build("/cartoon/demo").navigation();
        ARouter.getInstance().build("/read/main").navigation();
    }
}

File: app/src/main/java/club/hutcwp/lifeutil/HomeActivity.kt
package club.hutcwp.lifeutil

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import me.hutcwp.cartoon.webp.ui.ComicActivity
import me.hutcwp.cartoon.webp.ui.LoadActivity
import me.hutcwp.cartoon.webp.ui.MainActivity


class HomeActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)
//        startActivity(Intent(this, MainActivity::class.java))
        startActivity(Intent(this, LoadActivity::class.java))
//        startActivity(Intent(this, ComicActivity::class.java))



        finish()
    }
}
